### PR TITLE
docs: flow apps

### PR DESCRIPTION
**Pull request by razgraf**
This PR deals with the `apps`, `apis` and `support` sections of the docs, post-flow.

- [x] New technical guides tree-of-contents with references for other guides in the docs
- [x] Updated features menu and images for the vesting use-case
- [x] Updated images for drops, explanation re. airstreams vs. instant
- [x] New images and descriptions for payments
- [x] Refresh APIs structure (Lockup, Lockup Drops, Flow)
- [x] Update docs and app routes in guides

It also:

- Fixes formatter.js to abide by the new folder structure

### CODE REVIEW

File: docs/api/lockup/02-endpoints.mdx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+transitioned our subgraphs to either the Studio or The Decentralized Network (see
+[supported networks](https://thegraph.com/docs/en/developing/supported-networks/) and our endpoints below).
+
+:::
+
+### Endpoints
+
+| Chain            | Explorer                                                    | Studio[^2]                           | Decentralized Network[^1]                  |
+| ---------------- | ----------------------------------------------------------- | ------------------------------------ | ------------------------------------------ |
+| Ethereum         | [sablier-v2][v2-explorer-ethereum]                          | [Studio][v2-studio-ethereum]         | [De. Network][v2-network-ethereum]         |
```
Comment (smol-ninja) [inline docs/api/lockup/02-endpoints.mdx:28]: We should rename it to `sablier-lockup`. Wdyt?

**Review by smol-ninja — CHANGES_REQUESTED** [review]
There are mentions of `Sablier V2` at a few places. We should rename it to either `Sablier Lockup` or `Sablier Lockup V2`. Let me know which one you like @razgraf and I can make the change myself while reviewing this PR.

**Comment by PaulRBerg** [comment]
> There are mentions of Sablier V2 at a few places. We should rename it to either Sablier Lockup or Sablier Lockup V2

I reckon we should go with `Sablier Lockup` to completely get rid of the V2 versioning. Right?

File: docs/guides/lockup/03-gas-benchmarks.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ view the code that generates these benchmarks.
 | `renounce`                                                  | 19,381    |
 | `createWithDurations` (Broker fee set) (cliff not set)      | 129,276   |
 | `createWithDurations` (Broker fee not set) (cliff not set)  | 113,680   |
-| `createWithDurations` (Broker fee set) (cliff set)          | 138,071   |
+| `createWithDurations` (Broker fee set) (cliff set)          | 138,71   |
```
Comment (smol-ninja) [inline docs/guides/lockup/03-gas-benchmarks.md:30]: The formatter seems to have been formatting it incorrectly.

**Comment by smol-ninja** [comment]
Great work @razgraf. Other than the above two points, all looks good to me. Feel free to merge this PR after you have addressed them.

File: docs/api/01-overview.mdx [hunk 3]
```diff
@@ -11,78 +11,125 @@ import LinkPreview from "@site/src/components/LinkPreview";
...
-### The Graph's Subgraphs
+:::tip Playground
 
-[The Graph](https://thegraph.com/) has been the vendor of choice for the EVM space for the past few years. We've been
-integrating The Graph since 2019 with Sablier V1. For Sablier Lockup, we designed a special set of feature-oriented
-subgraphs used not only by us (in the official apps) but by 3rd party services (e.g. snapshot) and integrators alike.
+To preview the GraphQL APIs for subgraphs and indexers you can plug them into
+[Hasura's](https://cloud.hasura.io/public/graphiql) online explorer.
+[Here](https://cloud.hasura.io/public/graphiql?endpoint=https://api.studio.thegraph.com/query/57079/sablier-v2/version/latest)'s
+an example setup for the `sablier-v2` mainnet subgraph inside Hasura's explorer.
```
[the remark this answers is not in the document]
Comment (razgraf) [reply docs/api/01-overview.mdx:33]: Assuming this was feedback for all subgraph names, it's a tricky one, as I'm not 100% sure what happens on name changes for subgraphs. It's important that they are kept the same or point in the same directions, as we have external utilities (snapshot, defillama) and integrators relying on those endpoints.

Context: all our current "lockup" subgraphs are called `sablier-v2-*`, flows are `sablier-v2-flow-*` and drops `sablier-v2-ms-*`.

I've just checked and it looks like I can update the name of the subgraph and the `slug` updates as well. Next I'll ask in their discord if the old `slug` will redirect to the new one automatically. If yes, we can update, if no, we need to keep the current naming scheme.

File: docs/guides/lockup/03-gas-benchmarks.md (hunk 2 above)
Comment (razgraf) [reply docs/guides/lockup/03-gas-benchmarks.md:30]: Oh, good catch! I've updated the route to gas-benchmarks and the script ran on top of it.

Looking at the code now, I think the regex was only configured for non-padded values, meaning `138,071` was treated as `138` and `071`.

https://github.com/sablier-labs/docs/blob/9eedfa220743e9b21d9855f9fe8b6c73e0436545/formatter.js#L5C52-L5C62

File: docs/api/01-overview.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (smol-ninja) [reply docs/api/01-overview.mdx:33]: Thank you for your reply. In that case, should we be more explicit about it in the API's overview page? If I understand correctly, these are the reference:

1. `sablier-v2-*` means Lockup
2. `sablier-v2-flow-*` means Flow

If so, we can make a not at the top of the API section overview.

[the remark this answers is not in the document]
Comment (razgraf) [reply docs/api/01-overview.mdx:33]: Screenshot 2024-10-21 at 13 52 45

**Pull request merged**